src/Types.hs

```diff
@@ -322,6 +322,11 @@ phaseName = \case
     PhaseVoting     _ -> "Abstimmungsphase"
     PhaseResult       -> "Ergebnisphase"
 
+followingPhase :: Phase -> Phase -> Bool
```

Comment:
In that case the parameters should be swapped, but I am ok with that version too.